integration_tests/tests/forseti/controls/explain.rb

```diff
@@ -0,0 +1,375 @@
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_roles --prefix roles/storage").stdout).to match /roles\/storagetransfer.user/
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_roles --prefix roles/storage").stdout).to match /roles\/storagetransfer.viewer/
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_roles --prefix roles/storage").stderr).to eq ""
+            end
+        end
+
+        describe "List IAM roleAdmin permissions" do
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_permissions --roles roles/iam.roleAdmin").stdout).to match /roles.create/
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_permissions --roles roles/iam.roleAdmin").stdout).to match /roles.delete/
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_permissions --roles roles/iam.roleAdmin").stdout).to match /roles.get/
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_permissions --roles roles/iam.roleAdmin").stdout).to match /roles.list/
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_permissions --roles roles/iam.roleAdmin").stdout).to match /roles.undelete/
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_permissions --roles roles/iam.roleAdmin").stdout).to match /roles.update/
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_permissions --roles roles/iam.roleAdmin").stdout).to match /resourcemanager.projects.get/
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_permissions --roles roles/iam.roleAdmin").stdout).to match /resourcemanager.projects.getIamPolicy/
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_permissions --roles roles/iam.roleAdmin").stderr).to eq ""
+            end
+        end
+
+        describe "List IAM storage.Admin permissions" do
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_permissions --roles roles/storage.Admin").stdout).to match /firebase.projects.get/
```

Comment:
Can all the `list_permissions`  matching be consolidated?  Wouldn't the output be all the permissions instead of just one?

Reply:
The only other way to consolidate the tests is to use a regular expression. The complex regex impacts the readability and is harder to maintain when new assertions needs to be added.